docs/docs/widgets/pdf.md

```diff
@@ -63,24 +63,26 @@ To display instructional text when a user hovers over the PDF component, add som
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Devices
 
 | <div style={{ width:"100px"}}> Property   </div>       | <div style={{ width:"100px"}}>  Description           </div>                    | <div style={{ width:"135px"}}>  Expected Value </div>   |
-| :-------------- | :---------------------------------------- | :------------------ |
-| Show on desktop | Display the component in Desktop view. | Programmatically determine the value by clicking on `fx` or use the toggle switch |
-| Show on mobile  | Display the component in Mobile view.  | Programmatically determine the value by clicking on `fx` or use the toggle switch |
+| :-------------- | :------------------------------------------- | :------------------ |
+| Show on desktop | Makes the component visible in desktop view. | You can programmatically determine the value by clicking on **fx** or use the toggle switch. |
```

Comment:
Hey @dorkydhruv can you please update the content of the expected value column from the reference given to you in the issue?
Please do it for both the rows and both versions.

Reply:
@RudraPratik30 I have changed the expected value column's content as in the reference. I took liberty in fixing some spacings (prettier).
Let me know if we still need any changes.